.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Features Description
+1. Feature explanation
+   - Feature specification if you want to add
+
+### Depends On:
+- [ ] (Package PR#Number)[URL]
```

Comment:
I think this is situational. I suggest commenting it

Reply:
I do not know, I like having the most stuff here as convenient,  If I do not use I just erase, easier than writing it again. We can use comments instead of erasing lines, we uncomment what we need, but I do not know if this will be easier with the browser

Reply:
OK. I agree that deleting is better them uncommenting in the browser